tooltester/src/test/java/io/dockstore/tooltester/client/cli/ClientTest.java

```diff
@@ -5,10 +5,7 @@
 import com.beust.jcommander.ParameterException;
 import io.dockstore.tooltester.helper.JenkinsHelper;
 import io.dockstore.tooltester.helper.PipelineTester;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
+import org.junit.*;
```

Comment:
Configure your IDE to avoid these star imports. I think there's a setting to individually specify imports up to 999 from the same package or something. https://medium.com/@aayushsharda/why-is-it-better-to-avoid-in-import-statements-in-java-ff9ceaca8aeb

Reply:
I have removed the start imports from this PR and corrected my IDE.